app/helpers/mailer_helper.rb

```diff
@@ -30,12 +30,10 @@ def add_reason(text, reason)
   end
 
   def step_status_icon(step)
-    if step.status == "actionable"
-      "emails/numbers/icon-number-" + (step.position - 1).to_s + ".png"
```

Comment:
since we are not longer using these numbered icons can we remove them from source control?